server/repository/src/db_diesel/report.rs

```diff
@@ -21,6 +22,25 @@ pub struct Report {
     pub argument_schema: Option<FormSchema>,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct ReportMetaData {
```

Comment:
our domain object are composites atm, we don't really copy them (a bit less mapping and less to change when new fields are added)

Reply:
So should just have Version as another key (of type Version)